jeremy/README.md

```diff
@@ -0,0 +1,21 @@
+# Lab 5 Forms
+
+**Author**: Jeremy Beck
+**Version**: 1.0.0 (increment the patch/fix version number up if you make more commits past your first submission)
+
+## Overview
+Create a form to allow the user to input custom articles that will then be exported as JSON to disply on the index.html.
+
+## Getting Started
+Review user stories then complete to-do's. Test functionality.
+
+## Architecture
+HTML, JS, Handlebars.js, jQuery, JSON
+
+## Change Log
```

Comment:
make sure you are previewing your markdown files, these 2 lines need to be a list or have a line break as they are appearing on the same line